ibis/backends/bigquery/__init__.py

```diff
@@ -457,19 +457,29 @@ def create_table(
             raise NotImplementedError(
                 "BigQuery backend does not yet support overwriting tables"
             )
-        if schema is not None:
-            table_id = self._fully_qualified_name(name, database)
-            table = bq.Table(table_id, schema=BigQuerySchema.from_ibis(schema))
-            self.client.create_table(table)
-        else:
+
+        if isinstance(obj, ir.Table) and schema is not None:
+            if not schema.equals(obj.schema()):
+                raise com.IbisTypeError(
+                    """Provided schema and Ibis table schema are incompatible.
+                    Please align the two schemas, or provide only one of the two arguments."""
```

Comment:
How does this render in the terminal given the preservation of whitespace?

Reply:
Very badly!

Reply:
pushed up a fix